README.md

```diff
@@ -110,6 +110,31 @@ bundle exec rake test:all
 bundle exec rake
 ```
 
+## Internal documentation
+
+### Interacting with unstable position data
+
+We use data from the `unstable/positions.csv` file in a legislature's directory to display cabinet positions on the website. Because this data is unstable there currently aren't methods in [the `everypolitician` gem](https://github.com/everypolitician/everypolitician-ruby) for interacting with it, so viewer-sinatra has some of its own classes for dealing with it. These classes live in the `lib/everypolitician_extensions.rb` file and are documented below.
```

Comment:
not specifically an issue here, but just for explicit clarity, it's not only because the file lives in `unstable/` that we can't interact with it from the gem — it's that the way we store this data *will* change. The `positions.csv` is just a very quick hack to let us get up and running — we still need to work out how exactly we're going to store this sort of information, and the answer may not even be within the legislature at all.